lib/features/caching/caching_manager.dart

```diff
@@ -118,6 +118,13 @@ class CachingManager {
     ], eagerError: true);
   }
 
+  Future<void> clearMailboxCache() {
+    return Future.wait([
+      _mailboxCacheClient.clearAllData(),
+      if (PlatformInfo.isMobile) clearAllFileInStorage(),
```

Comment:
Please remove it, we only need remove mailbox cache and state of mailbox

Reply:
so just this ?

```
Future<void> clearMailboxCache() {
    return Future.wait([
      _mailboxCacheClient.clearAllData(),
      _stateCacheClient.clearAllData()
    ], eagerError: true);
  }
  ```

Reply:
stateCache store the data via `tuppleKey`. So we only to clear state of mailbox. Please try with
```
_stateCacheClient.deleteItem(StateType.mailbox.getTupleKeyStored(accountId, session.username))
```

Reply:
okie so please 
```
Future<void> clearMailboxCache() {
  return Future.wait([
    _mailboxCacheClient.clearAllData(),
  ], eagerError: true);
}
```

Reply:
done !